cypress/e2e/specialFeatures/atiAnalytics/assertions/index.js

```diff
@@ -103,78 +137,186 @@ export const assertPageView = ({
   });
 };
 
-const getViewClickDetailsRegex = ({ contentType, component, pageIdentifier }) =>
-  new RegExp(
-    `PUB-\\[${contentType}(.*)?\\]-\\[${component}(.*)?\\]-\\[(.*)?\\]-\\[(.*)?\\]-\\[${pageIdentifier}\\]-\\[(.*)?\\]-\\[(.*)?\\]-\\[(.*)?\\]`,
-    'g',
+const assertCPVModelViewEvent = ({
```

Comment:
Is it overkill to rename this? 
```suggestion
const assertClickPerViewModelViewEvent = ({
```

Reply:
Not at all.
I've update the view and click functions to use "Click Per View" in lieu of "CPV"

Reply:
Thanks. I know it will probably get deleted soon, but it might need to linger for longer because lite still uses this model 😬